android/sdk/src/main/java/com/taobao/weex/utils/batch/BatchOperationHelper.java

```diff
@@ -31,7 +32,7 @@
 
 
   private BactchExecutor mExecutor;
-  private ArrayList<Runnable> sRegisterTasks = new ArrayList<>();
+  private CopyOnWriteArrayList<Runnable> sRegisterTasks = new CopyOnWriteArrayList<>();
```

Comment:
`CopyOnWriteArrayList` is bad for performance in high frequency invoke situation. Be careful